apis/r/tests/testthat/test-SOMAArrayReader-Iterated.R

```diff
@@ -94,12 +94,15 @@ test_that("Iterated Interface from SOMA Classes", {
     ## parameterize test
     test_cases <- c("data.frame", "sparse")
 
+    # The read_complete et al. in this test case are designed to be verified
+    # against 16MB buffer size, and the particular provided input dataset.
+    ctx <- SOMATileDBContext$new(c(soma.init_buffer_bytes=as.character(16777216)))
+
     for (tc in test_cases) {
         sdf <- switch(tc,
-                      data.frame = SOMADataFrame$new(uri, internal_use_only = "allowed_use"),
-                      sparse = SOMASparseNDArray$new(uri, internal_use_only = "allowed_use"))
+                      data.frame = SOMADataFrameOpen(uri, tiledbsoma_ctx = ctx),
+                      sparse = SOMASparseNDArrayOpen(uri, tiledbsoma_ctx = ctx))
```

Comment:
Switch to factory method is good, the test was likely written before we added these.

In the interim @mojaveazure and I had huddled too and see that the create step has the entry for a ctx object so this seems like the best approach.